packages/gatsby-theme-bulmaio/gatsby-config.js

```diff
@@ -13,6 +13,35 @@ module.exports = {
                 maxWidth: 1080
             }
         },
+        {
+            resolve: `gatsby-plugin-sitemap`,
+            options: {
+                createLinkInHead: true,
+                query: `
+                    {
+                      site {
+                        pathPrefix
+                      }
+            
+                      allSitePage {
+                        nodes {
+                          path
+                        }
+                      }
+                  }`,
+                resolveSiteUrl: ({site, allSitePage}) => {
+                    return 'https://www.jetbrains.com'
```

Comment:
Should this /space/guide ?

Reply:
No, updates automatically. Should work for Space, .net, ws, py, ...